userguide/virtualmachines.rst

```diff
@@ -51,7 +51,10 @@ shows a list of installed virtual machines.
    Virtual Machines
 
 
-The |ui-options| menu has options for controlling and modifying VMs:
+*Name*, *State*, and :guilabel:`Autostart` are displayed on the
```

Comment:
Why is Autostart a guilabel different from the other two values?

Reply:
Because Autostart is a checkbox that can be clicked in the list of VMs